src/i18n/zh-cn/ui.ts

```diff
@@ -1,47 +1,64 @@
-import { UIDictionary } from '../translation-checkers';
-
-export default UIDictionary({
-    'a11y.skipLink': '跳转到内容',
-	'navbar.a11yTitle': '导航顶部',
-    // Site settings
+export default {
+	'a11y.skipLink': '跳转至内容',
+	'navbar.a11yTitle': '顶部',
+	// Site settings
 	'site.title': 'Astro 文档',
-	'site.description': '用更少的客户端 JavaScript 构建更快的站点。',
+	'site.description': '使用更少的客户端 Javascript 构建更快的网站。',
 	'site.og.imageSrc': '/default-og-image.png?v=1',
-	'site.og.imageAlt': '星辰浩瀚的太空中的 astro 标志，右侧前景中漂浮着一颗紫色的土星状行星。',
+	'site.og.imageAlt': '漫漫星空中的 Astro 图标，右侧前景中漂浮着紫色土星一般的星球',
 	// Left Sidebar
-	'leftSidebar.a11yTitle': '站点导航',
+	'leftSidebar.a11yTitle': '主要内容',
 	'leftSidebar.learnTab': '学习',
 	'leftSidebar.referenceTab': '参考',
-	'leftSidebar.noTranslations': '没有找到翻译内容',
-	'leftSidebar.viewInEnglish': '查看英文版本',
+	'leftSidebar.noTranslations': '未找到翻译',
+	'leftSidebar.viewInEnglish': '查看英文版文档',
 	// Right Sidebar
-	'rightSidebar.a11yTitle': '目录',
-	'rightSidebar.onThisPage': '在本页内',
-	'rightSidebar.overview': '概述',
-	'rightSidebar.more': '更多',
-	'rightSidebar.editPage': '编辑本页内容',
-	'rightSidebar.translatePage': '翻译本页内容',
-	'rightSidebar.joinCommunity': '加入社群',
+	'rightSidebar.a11yTitle': '次要内容',
+	'rightSidebar.onThisPage': '此页上',
+	'rightSidebar.overview': '概览',
+	'rightSidebar.community': '社群',
+	'rightSidebar.joinDiscord': '加入我们的 Discord 社群',
+	'rightSidebar.readBlog': '阅读博客',
+	'rightSidebar.openCollective': '参与众筹',
+	'rightSidebar.contribute': '贡献',
+	'rightSidebar.editPage': '编辑此页',
+	'rightSidebar.translatePage': '翻译此页',
+	'rightSidebar.github': 'GitHub 的 Astro 文档',
 	// `<ThemeToggleButton>` acessibility labels
-	'themeToggle.useLight': '使用浅色主题',
-	'themeToggle.useDark': '使用深色主题',
-	'rightSidebar.github': 'GitHub 上的 Astro 文档',
+	'themeToggle.useLight': '使用亮色主题',
+	'themeToggle.useDark': '使用暗色主题',
 	// Used in previous/next page links at the bottom of pages
 	'articleNav.nextPage': '下一页',
 	'articleNav.prevPage': '返回',
 	// Used in `<Since>`: Added in: v0.24.0 [NEW]
-	'since.addedIn': '增加于：',
-	'since.new': '新',
+	'since.addedIn': '添加于：',
+	'since.new': '新版本',
 	// Installation Guide
-	'install.autoTab': '自动化命令行',
-	'install.manualTab': '手动设置',
+	'install.autoTab': '自动 CLI',
+	'install.manualTab': '手动安装',
+	// `<DeployGuidesNav>` vocabulary
+	'deploy.sectionTitle': '部署指南',
+	'deploy.altSectionTitle': '更多部署指南',
+	'deploy.filterLabel': '按部署类型筛选',
+	'deploy.ssrTag': 'SSR',
+	'deploy.staticTag': '静态',
 	// `<ContributorList>` fallback text
 	'contributors.seeAll': '查看所有贡献者',
 	// Fallback content notice shown when a page is not yet translated
-	'fallbackContent.notice': '此页暂不支持你的语言，我们将为你展示英文页面。你可以帮忙翻译它！',
-	'fallbackContent.linkText': '了解更多关于贡献的内容',
+	'fallbackContent.notice': '此页面尚未被翻译，我们将为您展示未翻译的英文版本。请帮助我们翻译此页！',
+	'fallbackContent.linkText': '了解如何参与贡献',
 	// 404 Page
-	'404.title': '未能找到',
-	'404.content': '该页面不在我们的星系中。',
-	'404.linkText': '返回主页',
-});
+	'404.title': '未找到',
+	'404.content': '这个页面不在我们的星系中。',
+	'404.linkText': '带我回家',
```

Comment:
```suggestion
	'404.content': '该页面不在我们的星系中。',
	'404.linkText': '返回主页',
```

Please stick to being as concise and objective as possible without compromising the fluency of your statement. Here, home refers not to home (家), which is an emotional thing, but to the home page of the website.